test/integration/default/default_spec.rb

```diff
@@ -41,3 +41,9 @@
   its('stdout') { should match /Username/ }
   its('exit_status') { should eq 0 }
 end
+
+describe command('sensuctl check list') do
```

Comment:
we should use the as json matcher as this does not really give us the guarantee that the `dad_jokes` handler was added to `cron` and not some other check.

Reply:
Fixed